src/NUnitEngine/nunit.engine.core/Internal/RuntimeFrameworks/NetCoreFrameworkLocator.cs

```diff
@@ -26,7 +26,7 @@ public static IEnumerable<RuntimeFramework> FindDotNetCoreFrameworks()
                     alreadyFound.Add(newVersion);
                     // HACK: Avoid Exception for an unknown version - see issue #1223
                     // Requires change in RuntimeFramework.GetClrVersionForFramework()
-                    if (newVersion.Major <= 7)
+                    if (newVersion.Major <= 8)
```

Comment:
As nothing is really done with this, we could as well make this the default case for all Major > 5

Reply:
This also sounds like a good idea.

Reply:
Agree, will fix too.